### ISSUE TITLE
Menu in menu issue for versioned docs

### ISSUE THREAD

**Issue by jamtur01** [prometheus/docs#906]
There's a nested, versioned menu inside the old menu structure - Image is the best way to describe it:

<img width="1026" alt="screenshot 2017-11-11 08 18 40" src="https://user-images.githubusercontent.com/4365/32687272-16c9ec24-c6b9-11e7-8653-20ac8c6a29a8.png">

I'm not sure if this is intentional? (or if this should be log'ed on the main Prometheus repo)

**Comment by grobie** [comment]
With the introduction of the versioned docs, we had to introduce a second menu level. Is that what you find unexpected? The styling can be certainly better, I'm all but a designer ...

**Comment by jamtur01** [comment]
So a few things:

1. Why not surface the version selection at a higher level (or pop the drop down selection OUTSIDE of the  menu c.f. 
<img width="386" alt="screenshot 2017-11-11 18 17 13" src="https://user-images.githubusercontent.com/4365/32691227-95be9e14-c70c-11e7-9654-32dea3f4d1ea.png">
?)

2. It's odd the second menu level is called Prometheus - you're already IN the Prometheus docs. 
3. The Getting Started exists in the Prometheus menu block whilst the First Steps guide (which is 2.0 based) is under Introduction and contains a lot of overlap/duplicate material.

**Comment by grobie** [comment]
Thanks for the comment!

1. The Prometheus project consists of many official parts, see https://github.com/prometheus. We'll eventually have versioned docs for many of them. The next one will be Alertmanager, see #857. Each has their own version and we need a dropdown for each.

2. See 1. If you have a better suggestion, let me know!

3. Yes, the getting started guide should be removed from Prometheus metrics, no need to version it. It'll be updated to cover the latest versions of all components.

If you have any ideas how to make this all clearer, please let us know!

**Comment by jamtur01** [comment]
Okay. Perhaps my confusion is that the Prometheus section only contains some of the Prometheus docs. Is there a roadmap to move more or all of them under that heading? If I'm just seeing this in transition perhaps it's moot?

**Comment by grobie** [comment]
It should contain all the metrics which are specific to the Prometheus server itself, and which should be versioned as they can change over time. It's totally possible that we have missed some parts, what would you expect to be part of the versioned docs?

**Comment by jamtur01** [comment]
Okay. I think some of Introduction, Concepts probably overlap. I am afraid I can't think of a better name for the section though. Thanks for clarifying!

**Comment by RichiH** [comment]
I agree with @jamtur01 that the current layout is unexpected and somewhat confusing. The Docker example looks nice and it could change depending on which component the user is sitting on.

This also touches the old discussion of having a "latest" to point to instead of being forced to always version links.

**Comment by grobie** [comment]
> This also touches the old discussion of having a "latest" to point to instead of being forced to always version links.

That's not the case?

**Comment by RichiH** [comment]
Erm, yes, you are right and I seem to need new glasses.

**Comment by grobie** [comment]
So your request is to move that version picker out to another place. I'm not a designer, if someone has a mockup where to move it, I'm happy to discuss.

I'm sceptical that trying to mirror existing layouts of pages only having a single version scheme will help to reduce confusion. A version picker which changes based on the context seems to be a lot less explicit. Given users will have certain expectations around how that UI works having it seen on single-scheme pages, will that really help to make things more clear?

**Comment by jan--f** [comment]
Hello from the bug scrub!

I think I agree with @grobie. Also it seems users have adjusted or at least aren't complaining anymore? I'll close this, please feel free to re-open if there is still a discussion to be had.
